frontend/src/queries.js

```diff
@@ -89,7 +89,14 @@ export const fetchTournaments = async () => {
     headers: { "Content-Type": "application/json" },
     credentials: "same-origin"
   });
-  return await response.json();
+  let tournaments = await response.json();
+  function liveTournamentsFirst(a, b) {
```

Comment:
nit: Instead of this function definition make use of arrow functions and write it inline inside sort() function itself?

Reply:
yeah sure, i'll fix that

Reply:
Also, shouldn't the compare function deal with at least the case of `b` being live, but not a? And also may be make explicit the return values in the other cases -- where both are live, or both are not live?